modules/cluster-logging-manual-rollout-rolling.adoc

```diff
@@ -26,63 +26,39 @@ To perform a rolling cluster restart:
 $ oc project openshift-logging
 ----
 
-. Use the following command to extract the CA certificate from Elasticsearch and write to the *_admin-ca_* file:
+. Perform a shard synced flush using the {product-title} link:https://github.com/openshift/origin-aggregated-logging/tree/master/elasticsearch#es_util[*es_util*] tool to ensure there are no pending operations waiting to be written to disk prior to shutting down:
 +
 [source,terminal]
 ----
-$ oc extract secret/elasticsearch --to=. --keys=admin-ca
-----
-+
-[source,terminal]
-----
-admin-ca
-----
-
-. Perform a shard synced flush to ensure there are no pending operations waiting to be written to disk prior to shutting down:
-+
-[source,terminal]
-----
-$ oc exec <any_es_pod_in_the_cluster> -c elasticsearch -- curl -s --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key -XPOST 'https://localhost:9200/_flush/synced'
+$ oc exec <any_es_pod_in_the_cluster> -c elasticsearch -- es_util -s --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key -XPOST 'https://localhost:9200/_flush/synced'
 ----
 +
 For example:
 +
 [source,terminal]
 ----
-$ oc exec -c elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6 -- curl -s --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key -XPOST 'https://localhost:9200/_flush/synced'
+$ oc exec -c elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6 -c elasticsearch -- es_util -s --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key -XPOST 'https://localhost:9200/_flush/synced'
 ----
 
 . Prevent shard balancing when purposely bringing down nodes using the {product-title}
 link:https://github.com/openshift/origin-aggregated-logging/tree/master/elasticsearch#es_util[*es_util*] tool:
 +
 [source,terminal]
 ----
-$ oc exec <any_es_pod_in_the_cluster> -c elasticsearch -- es_util --query=_cluster/settings -XPUT 'https://localhost:9200/_cluster/settings' -d '{ "transient": { "cluster.routing.allocation.enable" : "none" } }'
+$ oc exec <any_es_pod_in_the_cluster> -c elasticsearch -- es_util --query=_cluster/settings -XPUT 'https://localhost:9200/_cluster/settings' -d '{ "persistent": { "cluster.routing.allocation.enable" : "primaries" } }'
```

Comment:
oc exec <any_es_pod_in_the_cluster>  -c elasticsearch -- es_util  '--query=_cluster/settings' -XPUT  -d '{ "transient": { "cluster.routing.allocation.enable" : "none" } }'

Reply:
`es_util --query="_cluster/settings" -XPUT -d '{"persistent": ... '`

Reply:
this needs to be both "persistent" and "primaries"
"transient" and "none" is old and incorrect